clients/client-web/package.json

```diff
@@ -34,9 +34,11 @@
     "webpack-cli": "^3.2.3"
   },
   "dependencies": {
+    "taskcluster-lib-urls": "^12.0.0"
```

Comment:
Why is this one OK as a dependency while the others aren't?

Reply:
They're all ok as dependencies but treeherder has asked in the past to trim down the package size. They asked for the consumer to install their own `crypto-js`, `query-string`, `hawk` since they already use it in their app to do other stuff with it.